rest/import_test.go

```diff
@@ -334,3 +335,73 @@ func TestXattrDoubleDelete(t *testing.T) {
 	revId = body["rev"].(string)
 
 }
+
+func TestViewTombstoneRetrieval(t *testing.T) {
```

Comment:
Can you rename this to TestViewQueryTombstoneRetrieval?  As it stands it sounds like it's just viewing (reading) a tombstone.  Adding "ViewQuery" might call out that it's a "CouchbaseView"